arkouda_proxy_client/arkouda_proxy_client/arkouda_pb2_grpc.py

```diff
@@ -0,0 +1,66 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
+import grpc
+
+from . import arkouda_pb2 as arkouda__pb2
+
+
+class ArkoudaStub(object):
+    """Missing associated documentation comment in .proto file."""
+
+    def __init__(self, channel):
+        """Constructor.
+
+        Args:
+            channel: A grpc.Channel.
+        """
+        self.HandleRequest = channel.unary_unary(
+                '/arkouda.Arkouda/HandleRequest',
+                request_serializer=arkouda__pb2.ArkoudaRequest.SerializeToString,
+                response_deserializer=arkouda__pb2.ArkoudaResponse.FromString,
+                )
+
+
+class ArkoudaServicer(object):
+    """Missing associated documentation comment in .proto file."""
+
+    def HandleRequest(self, request, context):
+        """Missing associated documentation comment in .proto file."""
+        context.set_code(grpc.StatusCode.UNIMPLEMENTED)
+        context.set_details('Method not implemented!')
+        raise NotImplementedError('Method not implemented!')
+
+
+def add_ArkoudaServicer_to_server(servicer, server):
```

Comment:
This function name is a little confusing to me. I am assuming that the servicer being added here will be the one from the `Arkouda` class below and not the `ArkoudaServicer` class above since it is not implemented? Or am I missing something here?

Reply:
oh, that's code generated by the python protobuf compiler. To be honest I don't _totally_ understand the generated code myself. I'll dig in more and documen t in the future.

Reply:
Sounds good.